docs/build-integration.md

```diff
@@ -0,0 +1,38 @@
+---
+id: build-integration
+title: Polkadot Integration Guide
+sidebar_label: Integration Initiation
+---
+
+Welcome to the Polkadot integration guide. This guide will provide all the information you need to
```

Comment:
The word 'guide' is repeated twice, is there another word we could change it to instead?

Reply:
I think it's OK, better than "This will..."